src/jvm/test/resources/expected_logs/validation_function_failure.txt

```diff
@@ -23,8 +21,6 @@ The following interleaving leads to the error:
 | inc()    |
 | inc()    |
 | inc()    |
-| inc()    |
-| inc()    |
 | -------- |
 
 Detailed trace:
```

Comment:
It's unclear from the output when the validation function fails

Reply:
Sure, but can we do something with that? I suggest opening a separate issue and continuing the discussion there.

Reply:
Ok, I opened the issue #220